app/views/carriers/index.haml

```diff
@@ -0,0 +1,25 @@
+%h1 Carriers
+.row
+  .col-md-6.offset-md-3
+    %table.table.table-sm
+      %thead.thead-dark
+        %tr
+          %th Name
+          %th Gateway Address
+          %th
+          %th
+      %tbody
+        - if @carriers.present?
+          - @carriers.each do |carrier|
+            %tr
+              %td= carrier.name
+              %td= carrier.gateway_address
+              %td= link_to 'Edit', edit_carrier_path(carrier)
+              %td.text-right= button_to 'Destroy', carrier, method: :delete,
+                class: 'btn btn-danger', data: { confirm: 'Are you sure?' }
+        - else
+          %tr
+            %td{colspan: 7} No carriers to show.
+    .row
+      .col.text-right
+        = link_to 'New Carrier', new_carrier_url
```

Comment:
I think we slightly prefer `path` but I can't remember why. @werebus may remember

Reply:
Yeah, the rationale is pretty weak, but it amounts to: "you don't need protocol, host, and port here, so why add that extra information to the page?" In days of yore, making links on your wobsite absolute really mucked things up when you went to move your static files to a hew host, a new subdirectory, etc. That  _doesn't_ apply here, but it's the background for my opinion.